cross/poetry/Makefile

```diff
@@ -5,21 +5,17 @@ PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = https://files.pythonhosted.org/packages/source/p/$(PKG_NAME)
 PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
 
-BUILD_DEPENDS = cross/cffi
-DEPENDS =
+DEPENDS = cross/cffi
 
 HOMEPAGE = https://python-poetry.org/
 COMMENT  = Poetry helps you declare, manage and install dependencies of Python projects, ensuring you have the right stack everywhere.
 LICENSE  = MIT
 
-POST_INSTALL_TARGET = install_into_cross_env
+PRE_INSTALL_TARGET = install_into_cross_env
 
 include ../../mk/spksrc.python-module.mk
 
 .PHONY: install_into_cross_env
 install_into_cross_env: 
-ifneq ($(strip $(CROSSENV)),)
-	@. $(CROSSENV) \
-		&& build-pip install "$(PKG_NAME)==$(PKG_VERS)" \
-		&& pip install "$(PKG_NAME)==$(PKG_VERS)"
-endif
+	@. $(CROSSENV) && $(RUN) build-pip install "$(PKG_NAME)==$(PKG_VERS)" $(ADDITIONAL_WHEEL_BUILD_ARGS)
```

Comment:
poetry doesn't support python 2 so no need to worry here.
Additionally the $(ADDITONAL_WHEEL_BUILD_ARGS) enforces the "no build isolation".